src/tls_wrap.cc

```diff
@@ -53,6 +53,18 @@ TLSWrap::TLSWrap(Environment* env,
   node::Wrap(object(), this);
   MakeWeak(this);
 
+  //
+  //TODO: This is a temp workaround for the stashed persistent pointer in (object()) that is made weak.
+  //      The resulting hidden lifetime means we lose the reference for it in the event replay code. 
+  //      We will need to:
+  //        (1) Add a method to notify the engine that these pointers are being stashed (likely in the persistent code).
```

Comment:
Is this something that we should consider accounting for in NAPI? cc @jasongin @boingoing